### ISSUE TITLE
[COGNITION]: Breadcrumbs - Lists SHOULD announce the number of items in VoiceOver

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#9085]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our Breadcrumb component doesn't announce the number of list items in the UL. This is due to Safari + VoiceOver [removing list semantics](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html) when developers use `li { list-style-type: none; }` CSS.

VO screen reader users would benefit from knowing the number of list items, so their experience is on par with JAWS and NVDA. Proposed code snippet below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

**As a VoiceOver user, I want the following to be true:**
- [ ] I will hear the number of list items read aloud like "List, three items". This is helpful for knowing how deep in the information architecture I currently am.

## Possible Fixes (optional)
This fix will update the `<ul>` and the `<li>`. Line numbers are listed in each code diff.
### Unordered List
```diff
! Breadcrumbs.jsx#49

<ul
  className="row va-nav-breadcrumbs-list columns"
  id={breadcrumbListId}
+ role="list"
>
  {this.renderBreadcrumbLinks()}
</ul>
```

### List Item
```diff
! Breadcrumbs.jsx#L20

renderBreadcrumbLinks = () => {
  const children = React.Children.toArray(this.props.children);
  children.push(
    React.cloneElement(children.pop(), {
      'aria-current': 'page',
    }),
  );
- return children.map((child, i) => <li key={i}>{child}</li>);
+ return children.map((child, i) => <li key={i} role="listitem">{child}</li>);
};
```

## WCAG or Vendor Guidance (optional)
* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #240](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/240) via [**ZenHub**](https://www.zenhub.com/)
